pre_backup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# docker exec fare-invenio_db_1 chmod 777 /backup_fare_db
+# docker exec fare-invenio_es_1 chmod 777 /usr/share/elasticsearch/backup_fare_es
+
+chmod 777 ~/fare_shared_data
```

Comment:
Uhm... this is weird, I'd try to avoid 777ing the whole folders

Reply:
Sure, it was done to try the general behaviour, it will not be present when the PR will be merged